packages/itwin/tree-widget/src/components/trees/models-tree/ModelsTree.tsx

```diff
@@ -147,7 +153,21 @@ export function ModelsTree({
     };
   }, [filter, getModelsTreeIdsCache, onFeatureUsed, hierarchyConfiguration]);
 
-  const getFilteredPaths = getFocusedFilteredPaths ?? getSearchFilteredPaths;
+  const getFilterPaths = useMemo<VisibilityTreeProps["getFilteredPaths"]>(() => {
```

Comment:
This is very similar to the name of `getFilteredPaths` prop. Maybe this one should be renamed to something like `getFilteredPathsFromProps`?

Reply:
Renamed.